internal/http/services/owncloud/ocdav/propfind.go

```diff
@@ -775,7 +775,7 @@ func (s *svc) mdToPropResponse(ctx context.Context, pf *propfindXML, md *provide
 				// I tested the desktop client and phoenix to annotate which properties are requestted, see below cases
 				case "fileid": // phoenix only
 					if md.Id == nil {
-						propstatNotFound.Prop = append(propstatNotFound.Prop, s.newProp("oc:fileid", ""))
+						continue
```

Comment:
Do we really need to ignore it completely, even not putting it in the not found section?